front/app/modules/commercial/content_builder/admin/components/ContentBuilderMobileView/ContentBuilderMobileView.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Should the mobile toggle be included in the e2e test as well?

Reply:
Definitely - as mentioned in the description, test coverage should be improved. But let's do it in a separate PR.